app/services/state_file/direct_file_api_response_sample_service.rb

```diff
@@ -77,7 +77,7 @@ def load_samples
 
       %w[json xml].each do |file_type|
         samples = file_type == 'json' ? @json_samples : @xml_samples
-        StateFile::StateInformationService.active_state_codes.each do |us_state|
+        (StateFile::StateInformationService.active_state_codes + ["test"]).each do |us_state|
```

Comment:
This allows fixtures in the test directory to be loaded

Reply:
to limit scope deprecated personas are put into this test file. Maybe we need some sort of direction for the future of this folder